### ISSUE TITLE
Any update for new versions?

### ISSUE THREAD

**Issue by josearellano** [learningequality/ka-lite-installer-android#16]
I'm trying to build an apk with this information, but I get no results.

Could you please tell me if you have any update version of ka-lite-android?

**Comment by peperepito** [comment]
@aronasorman 
@jamalex 
Do you have any update for this project? I can't build the APK. If you have an updated APK, that will help me a lot too.

**Comment by 66eli77** [comment]
Hi, @peperepito 
This project is not in active development anymore. Currently, our android folks are working on the next generation ka-lite app, Kolibri, which is also part of our first Kickstarter project. Please come back a couple months later for updates, or you can subscribe to our [Insider Updates](https://learningequality.org/) to get the latest information.

**Comment by peperepito** [comment]
Thanks a lot @66eli77 for the information. Do you have any idea of release dates?
This app is going to be free?

Thanks!

**Comment by 66eli77** [comment]
@peperepito yes, the app is going to be free and is expected to be released later this year depending on how our Kickstarter project goes.
